qiskit_bot/notifications.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019
```

Comment:
How much do we care about this stuff being accurate?  I think Nature have a checker for the copyrights, which might be nice to use for consistency across Qiskit in the future.

Reply:
It honestly isn't really enforceable for anything. Also `(C)` is meaningless it's not actually a copyright symbol.  It's just busy work imo to keep updating things on every when we have a complete git log to show modification history. For source distribution without the git history outside of the copyright statement itself the year doesn't really matter too much. That being said I can just fix this easily enough

```suggestion
# (C) Copyright IBM 2022
```

Reply:
Yeah, on qutip I literally just removed all these notices for that reason (qutip/qutip#1579).  The licence in the repo root is sufficient if it's distributed with the package.